aspnetcore/host-and-deploy/azure-apps/troubleshoot.md

```diff
@@ -84,12 +84,12 @@ Many startup errors don't produce useful information in the Application Event Lo
 
 ### ASP.NET Core Module stdout log
 
-The ASP.NET Core Module *stdout log* is another log helpful in diagnosing startup failures. To enable and view stdout logs, perform the following steps:
+To enable and view stdout logs:
```

Comment:
I have a concern on this one: I don't think just dumping the reader into this other approach makes sense without saying *something* about what it is or why it would be done. I'll mod the language to drop "helpful" ... although ... one man's "spam" is another man's "conversational tone." lol

Reply:
OK - then say why it's helpful. 

>  "conversational tone."

the style guide prefers that, but damien and dan.roth prefer ref style.

Reply:
I ended up using a slight variation of your sentence. You'll see it on the next commit. It's better now ... and *not "helpful"* lol.